codegen/compiler/src/Quidditch/Target/QuidditchTarget.cpp

```diff
@@ -173,7 +184,20 @@ class QuidditchTargetBackend final : public IREE::HAL::TargetBackend {
     }
     modulePassManager.addPass(createMaterializeUserConfigsPass());
     FunctionLikeNest funcPassManager(modulePassManager);
-    funcPassManager.addPass(quidditch::createConfigureForSnitchPass);
+
+    // import any manually supplied tile sizes
+    if (targetOptions.importTiles != "") {
+      std::string errs;
+      quidditch::fillTileInfoTable(&targetOptions.tileInfo,
+                                   targetOptions.importTiles, errs);
+    }
```

Comment:
Import tiling schemes and populate the table _before_ creating the ConfigureTiles pass